src/routes/plans/+page.svelte

```diff
@@ -422,43 +428,66 @@
             />
           </Field>
 
-          <fieldset>
-            <DatePickerField
-              field={startTimeDoyField}
-              label="Start Time - YYYY-DDDThh:mm:ss"
-              name="start-time"
-              on:change={onStartTimeChanged}
-              on:keydown={updateDurationString}
-              use={[
-                [
-                  permissionHandler,
-                  {
-                    hasPermission: canCreate,
-                    permissionError,
-                  },
-                ],
-              ]}
-            />
-          </fieldset>
-
-          <fieldset>
-            <DatePickerField
-              field={endTimeDoyField}
-              label="End Time - YYYY-DDDThh:mm:ss"
-              name="end-time"
-              on:change={updateDurationString}
-              on:keydown={updateDurationString}
-              use={[
-                [
-                  permissionHandler,
-                  {
-                    hasPermission: canCreate,
-                    permissionError,
-                  },
-                ],
-              ]}
-            />
-          </fieldset>
+          {#if $plugins.time.enableDatePicker}
```

Comment:
Should this be handled in the `DatePicker` component itself so this check doesn't have to happen in potentially other places that utilize a `DatePicker`? Maybe a component that wraps around the `DatePicker` that has logic similar to this that decides to show a `DatePicker` or regular input field?

Reply:
Good catch. I actually implemented a `useFallback` for `DatePicker` for this very purpose and used it around the app except for here. Fixed.